### PR TITLE
 Fix includes(va).references(va) in rails 5.1

### PR DESCRIPTION
**Pull request by kbrock**
In rails 5.1, it was not properly aliasing the names of columns
after an `includes().references()`

The attributes were coming across as the alias name instead of the attribute name

### CODE REVIEW

File: spec/virtual_attributes_spec.rb [hunk 1]
```diff
@@ -983,4 +983,13 @@ def col2
     tc.save
     expect(tc.attributes.size).to eq(2)
   end
+
+  it "doesn't botch up the attributes when requiring", :with_test_class do
```
Comment (jrafanie) [inline spec/virtual_attributes_spec.rb]: referencing?  I'm not sure what you mean by requiring

File: lib/active_record/virtual_attributes/virtual_fields.rb [hunk 2]
```diff
@@ -187,13 +187,17 @@ def find_with_associations
         else
           recs = real.find_with_associations
         end
-        MiqPreloader.preload(recs, preload_values + includes_values) if includes_values
+
+        if includes_values
+          ActiveRecord::Associations::Preloader.new.preload(recs, preload_values + includes_values)
```
Comment (Fryguy) [inline lib/active_record/virtual_attributes/virtual_fields.rb]: :tada:  Glad you got rid of MiqPreloader here 😄

File: spec/virtual_attributes_spec.rb (hunk 1 above)
Comment (kbrock) [reply spec/virtual_attributes_spec.rb]: stupid `s/require/references/g`

**Comment by kbrock** [comment]
fixed spec text. let me know if there is anything else

**Pull request merged**